assets/app/view/game/entities.rb

```diff
@@ -34,7 +35,10 @@ def render
           ])
         end
 
-        children << h(:div, bank_owned.map { |c| h(Corporation, corporation: c) })
+        children << h(:div, [
+          h(Bank, game: @game, layout: :card),
+          *bank_owned.flat_map { |c| h(Corporation, corporation: c) },
```

Comment:
should this be map?

Reply:
remains from earlier dev stage, both should work